src/common/mfu_daos.c

```diff
@@ -1068,6 +1096,23 @@ int daos_connect(
                 goto bcast;
             }
 
+#ifdef HDF5_SUPPORT
+            /* need to wait until destination container is valid to set user
+             * attributes, preserve is only set true when destination is a DFS
+             * copy */
+            if (preserve) { 
+                /* deserialize and set the user attributes if they exist */
+                htri_t usr_attrs_exist = H5Lexists(hdf5.file, "User Attributes", H5P_DEFAULT);
+                if (usr_attrs_exist > 0) {
+                    rc = cont_deserialize_usr_attrs(&hdf5, *coh);
+                    if (rc != 0) {
+                        rc = 1;
+                        goto bcast;
+                    }
```

Comment:
For ``cont_deserialize_usr_attrs`` we set ``rc = 1``, but we do not set ``rc = 1`` for ``cont_deserialize_all_props`` above.  In both cases, I see they use ``rc`` to hold the return value from the function call, so I'm guessing either way is valid.  Can we make them consistent?